src/modules/logger/logger.cpp

```diff
@@ -565,6 +566,15 @@ void Logger::add_default_topics()
 	add_topic("vehicle_land_detected");
 }
 
+void Logger::add_calibration_topics()
+{
+	// Note: try to avoid setting the interval where possible, as it increases RAM usage
+
+	add_topic("sensor_gyro", 100);
+	add_topic("sensor_accel", 100);
+	add_topic("sensor_baro",100);
```

Comment:
To make this more useful, we should use the mean of the full rate data over a fixed low rate interval such as 1 second. Then we could use it for the allan variance plots for random walk noise fitting.